applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -700,6 +700,10 @@ void slm_at_receive(const uint8_t *buf, size_t len)
 			break;
 		}
 
+#if defined(CONFIG_SLM_UART_ECHO)
```

Comment:
SLM can handle multiple commands coming in a single UART buffer. And this loop is run once for each command. This printing would need to be removed out from the loop.

Reply:
Now it's out from the loop.